index.d.ts

```diff
@@ -10,13 +10,14 @@ import MockClient = require('./types/mock-client')
 import MockPool = require('./types/mock-pool')
 import MockAgent = require('./types/mock-agent')
 import mockErrors = require('./types/mock-errors')
+import ProxyAgent from './types/proxy-agent'
```

Comment:
The import statement does not align with the export footprint of ProxyAgent ([index.js#L17]).

[index.js#L17]: https://github.com/RafaelGSS/undici/blob/f4e177a23e9aa73cd277f58b363aa20dc16cf870/index.js#L17